src/blockchain/log-processors/market-migrated.ts

```diff
@@ -41,7 +41,7 @@ export function processMarketMigratedLog(db: Knex, augur: Augur, log: FormattedE
             db.select("popularity").from("categories").where({ category, universe: log.newUniverse }).asCallback((err: Error|null, categoriesRows?: Array<CategoriesRow>): void => {
               if (err) return callback(err);
               if (categoriesRows && categoriesRows.length) return callback(null);
-              db.insert({ category, universe: log.newUniverse }).into("categories").asCallback(callback);
+              db.insert({ category: category.toUpperCase(), universe: log.newUniverse }).into("categories").asCallback(callback);
```

Comment:
shouldn't be necessary, since the value is coming from the DB, but def doesn't hurt, and no reason to remove it

Reply:
yeah the goal was to just try and make sure, (also a attempt to potentially make it backward compatible, but based on our convo idk that it will be and we will probably need a full resync)